pkg/kubectl/cmd/edit.go

```diff
@@ -159,6 +160,7 @@ func RunEdit(f *cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args
 		NamespaceParam(cmdNamespace).DefaultNamespace().
 		FilenameParam(enforceNamespace, options.Recursive, options.Filenames...).
 		ResourceTypeOrNameArgs(true, args...).
+		ContinueOnError().
```

Comment:
I'd suggest splitting this part out of this pull.  `edit` is complicated enough that it would take a long time to review and I wouldn't want to hold the rest of the work on this.